### ISSUE TITLE
Setup should prepare CE to work with dynamic IP correctly

### ISSUE THREAD

**Issue by yurem** [GluuFederation/community-edition-setup#425]
Currently customers who install Gluu under AWS has problems with accessing it after install.
The one of the problem that setup put /etc/host with next line:
```
%(ip)s %(hostname)s
```

**Comment by mzico** [comment]
@yurem : For AWS, I found it best not to 'modify' any IP at the time of installation. setup.py take one IP_address which is best suited ( mostly internal IP of that AWS instance ). 
And as far as I can tell, internal_ip of AWS instance never change. 

To 'access' the login page from internet, if that hostname+ip is not DNS registered.. they need map that `external_ip_of_aws_instance [hostname_of_gluu_server]` in their own computer's /etc/hosts file.

**Comment by ganesh-at-wiw** [comment]
Hi @yurem ,

Here is the analogy I used similar to AWS:

AWS provides public name like this: ec2-56-38-174-34.us-west-2.compute.amazonaws.com(I'm using a fake name!). So, I used public name of our gluu server instance as: cf.gluu.org(This one is real).

Let's say, we want to access gluu-server at https://de.gluu.org. For this, I installed the gluu-server at our VM with hostname as: de.gluu.org and at internal IP.

Next, I did set de.gluu.org as CNAME to cf.gluu.org(This too is real). Which is same like AWS would expect some hostname like de.gluu.org to create CNAME to ec2-56-38-174-34.us-west-2.compute.amazonaws.com.

Finally, you access: https://de.gluu.org which is CNAME of cf.gluu.org and it's accessible.


Is this what we want? If yes, then we do not need to do anything.
